azure-pipelines.yml

```diff
@@ -30,16 +30,13 @@ stages:
       pool:
         vmImage: $(imageName)
       steps:
-      - script: npm install -g pnpm@6.19.0
-        displayName: install pnpm globally
-
-      - script: pnpm install
+      - script: npm install
```

Comment:
Since `package-lock.json` is published now, change to `npm ci`.

Reply:
changed